src/esm_runscripts/prepcompute.py

```diff
@@ -188,7 +183,7 @@ def modify_namelists(config):
         )
 
     if config["general"]["verbose"]:
-        print("::: end of namelist section\n")
+        logger.debug("::: end of namelist section\n")
```

Comment:
Don't you need to get rid of the `config["general"]["verbose"]`.

Reply:
Removed `config["general"]["verbose"]`